sphinx_fossasia_theme/static/fossasia.css_t

```diff
@@ -124,3 +124,35 @@ img {
     max-width: 100%;
     height: auto;
 }
+
+table {
+    border: 1px solid #ddd;
+    width: 100%;
+    max-width: 100%;
+    margin-bottom: 20px;
+}
+
+table > tbody > tr > td,
+table > tbody > tr > th,
+table > tfoot > tr > td,
+table > tfoot > tr > th,
+table > thead > tr > td,
+table > thead > tr > th {
+    border: 1px solid #ddd;
+}
+
+table > tbody > tr > td,
+table > tbody > tr > th,
+table > tfoot > tr > td,
+table > tfoot > tr > th,
+table > thead > tr > td,
+table > thead > tr > th {
```

Comment:
because it is like for td,th we doing a lot of work. if bootstrap follow this convention then it's fine :+1: